test/cases/wasm/js-incompatible-type/index.js

```diff
@@ -0,0 +1,11 @@
+it("should disallow exporting a func signature with result i64", function() {
+    return expect(import("./export-i64-result.wat")).rejects.toThrow(/invalid type/);
+});
+
+it("should disallow exporting a func signature with param i64", function() {
+    return expect(import("./export-i64-param.wat")).rejects.toThrow(/invalid type/);
```

Comment:
Anyway modules won't throw on import, because you don't use the export function.

Wrong imports will result in compile errors (which can't be catch at runtime -> `error.js` file).

Reply:
``` js
import { a } from "./export-i64-result.wat";
a();
```

This code should result in a compile error, because here we know that the export `a` is used.